src/rgw/rgw_sal.h

```diff
@@ -461,11 +587,6 @@ class User {
 
     /** Clone a copy of this user.  Used when modification is necessary of the copy */
     virtual std::unique_ptr<User> clone() = 0;
-    /** List the buckets owned by a user */
```

Comment:
Do we want the User to contain an rgw_owner now rather than an rgw_user?

Reply:
no, each `sal::User` still corresponds to a unique `rgw_user`

the `rgw_owner` variant is for resources like buckets/objects that can either be owned by a user or an account

when a user is 'owned' by an account, we track that relationship with `RGWUserInfo::account_id`